### ISSUE TITLE
Allow reordering connected methods from the Signals tab

### ISSUE THREAD

**Issue by Paul-Griffin-1517-1** [godotengine/godot-proposals#9088]
### Describe the project you are working on

Godot

### Describe the problem or limitation you are having in your project

Oftentimes you'll want to connect multiple methods to one signal. The order the methods are called depends on the order they are connected. However, when using the signal tab, there is no easy way to change this order without deleting a bunch of signals and re-adding them.

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

Something like a drag-and-drop option to change the order of signals would probably be the simplest solution.

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

See above.

### If this enhancement will not be used often, can it be worked around with a few lines of script?

Since it is a simple editor tweak, this should be possible with a few lines of script.

### Is there a reason why this should be core and not an add-on in the asset library?

This would improve usability of the signal system, which is a core Godot feature.

**Comment by KoBeWi** [comment]
You can easily reorder the connections by editing the tscn file from text editor.

**Comment by Mickeon** [comment]
Sometimes I connect methods to the wrong signal and changing them is a bit tedious. The ability to move them around makes perfect sense on paper, to me.

I remember this was written as a discussion, already...

**Comment by JoNax97** [comment]
Relying on the order in which a signal calls it's listeners sounds like a bad idea and I'm not sure it's something that Godot should endorse.. m

I support the flexibility in the UI for other use cases though

**Comment by Mickeon** [comment]
It's not just reordering, it's the ability to move connections around, even across different signals. The ability to "reorder" connections would come as a consequence to this.

**Comment by JoNax97** [comment]
Yeah, that's why I support the idea in general. I just wouldn't attract to much attention to the fact you can use it to reorder callbacks.

**Comment by ultrasuperpingu** [comment]
I have a more general problem on this: order of connections is not consistant. When I make sure the connections order is ok for me, after saving, the order change. Am I the only one having this issue ?
